backend/spec/features/admin/taxons_spec.rb

```diff
@@ -21,4 +21,30 @@
     click_button "Update"
     expect(page).to have_content("Taxon \"Shirt\" has been successfully updated!")
   end
+
+  scenario "admin should be able to remove a product from a taxon", js: true do
+    taxon_1 = create(:taxon, name: 'Clothing')
+    product = create(:product)
+    product.taxons << taxon_1
+
+    visit spree.admin_taxons_path
+    select_clothing_from_select2
+
+    find('.product').hover
+    find('.product .dropdown-toggle').click
+    # TODO: This should be a click_link "Delete from taxon"
+    page.execute_script("$('.js-delete-product').click()")
+    wait_for_ajax
+
+    visit current_path
+    select_clothing_from_select2
+
+    expect(page).to have_content("No results")
+  end
+
+  def select_clothing_from_select2
+    page.execute_script %Q{$('#s2id_taxon_id').select2('open')}
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.